source/ranges.tex

```diff
@@ -3394,6 +3399,42 @@
 The expression \tcode{C | D} is well-formed if and only if
 the initializations of the state entities of \tcode{E} are all well-formed.
 
+\pnum
+Given an object \tcode{t} of type \tcode{T}, where
+\begin{itemize}
+\item
+\tcode{t} is a unary function object that accepts a \libconcept{range} argument,
+\item
+\tcode{T} models \tcode{\libconcept{derived_from}<range_adaptor_closure<T>>},
+\item
+\tcode{T} has no other base classes of type \tcode{range_adaptor_closure<U>} for any other type \tcode{U}, and
+\item
+\tcode{T} does not model \libconcept{range}
+\end{itemize}
+then the implementation ensures
```

Comment:
(I'm not wild about the use of "then" here, which is trying to make it sound as if the sentence had started with "If t is an object of type T...". Really we should drop the "then" and have a comma after the preceding item. But that's how it was given to us.)